### DISCUSSION TITLE
function {{random 0 100}} does not work.

### DISCUSSION THREAD

**Discussion by AndRu-v1** [Budibase/budibase#7631] [Help]
connected the rest API module , in the section
Headers I can't write a formula to the value of the field. I want the field value to be random. the built-in function {{random 0 100}} does not work.
![image](https://user-images.githubusercontent.com/92476098/188423445-40c1754e-f9e3-4e70-bf44-a52beb227259.png)

It would be nice if if the value of the field is a formula, then when you hover over the formula, the result of its execution would be shown, as it is done in retool.com
![image](https://user-images.githubusercontent.com/92476098/188423574-7d503141-8c68-426e-bbdc-de0973672944.png)

**Comment by melohagan** [comment]
Hi @AndRu-v1 

You can pass bindings through to your REST query from the Design section: https://docs.budibase.com/docs/rest-bindings#data-provider-applications-provider
